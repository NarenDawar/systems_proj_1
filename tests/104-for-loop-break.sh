
for a in 1 2 "break" 3 4 "done"
do
    echo $a
    [ $a = "break" ] && break
done
