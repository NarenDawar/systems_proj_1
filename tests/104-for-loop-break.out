1
2
break
